src/parser/csql_grammar.y

```diff
@@ -14662,7 +14662,16 @@ opt_orderby_clause
 
 			if (stmt)
 			  {
-			    stmt->info.query.order_by = order = $5;
+                            PT_NODE *n = NULL;
+
+                            stmt->info.query.order_by = order = n = $5;
+
+                            while (n)
+                              {
+                                resolve_alias_in_expr_node (n, stmt->info.query.q.select.list);
```

Comment:
stmt can be any of the query type nodes. it could access wrong union type if node_type of stmt is not PT_SELECT. So unexpected behavior can occur.